microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Build PWA-driven Widgets
+description: How to build lightweight PWA-driven Widgets that integrate in the operating system widget dashboard.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: pwa
+ms.date: 08/01/2022
+---
+# Build PWA-driven Widgets
+
+Native applications can expose information and let users perform tasks via widgets in various operating system's widget dashboards. Examples of this include Android Home Screen Widgets, macOS Dashboard and Today Panel Widgets, the Apple Touch Bar, Samsung Daily Cards, Mini App Widgets, smart watch app companions.
+
+On Windows 11, Widgets can be displayed from the Taskbar.
+
+![Widgets on Windows 11](../media/windows11-widgets.png) 
+
+Progressive Web Apps (PWAs) can also declare Widgets, update them, and handle user interactions within them.
+
```

Comment:
```suggestion
![Experimental feature](../../media/2020/06/experimental-tag-14px.msft.png)
```
example of that png in a non-Whatsnew dir:
https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/inspect-styles/edit-fonts